openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -119,23 +121,27 @@ def _imprint(self, tool, data):
 
     def _update_tool_with_data(self, tool, data):
         """Update tool node name and output path based on subset data"""
-        if "subset" not in data:
-            return
 
-        original_subset = tool.GetData("openpype.subset")
+        if data.get("subset") is None:
```

Comment:
Maybe we could do the variable assignation before the if, so we only access the dict ones:
```py
subset = subset = data.get("subset")

if subset is None:
    self.log.warning("No subset found for _update_tool_with_data")
    return
```